WordPress/src/main/java/org/wordpress/android/WPWellSqlConfig.kt

```diff
@@ -0,0 +1,40 @@
+package org.wordpress.android
```

Comment:
Can we add this whole class into folder `/src/debug/java/org/wordpress/android`? This way we'd be making use of the `productFlavors` feature and would make 100% sure the code is not even bundled within a product version.

Then, you can add a stub class in `/src/main/java/org/wordpress/android/WPWellSqlConfig.kt` that just calls the superclass `WellSqlConfig` implementation.

Reply:
> Note: for extra precautions, we could move this to WordPressDebug.java

Reply:
> Can we add this whole class into folder `/src/debug/java/org/wordpress/android`

Done in f73652e